components/do_verify_email/do_verify_email.jsx

```diff
@@ -5,7 +5,11 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import ErrorStore from 'stores/error_store.jsx';
+import {trackEvent} from 'actions/diagnostics_actions.jsx';
+import {updateUser} from 'actions/user_actions.jsx';
```

Comment:
Instead of importing the action here, can you import it directly from mattermost-redux in the `index.js` container and pass it in as a prop? Similar to how `verifyUserEmail` is passed in?

Most of the actions in `actions/user_actions.jsx` are legacy and really just wrappers around the mattermost-redux actions

This document might have some helpful information https://developers.mattermost.com/contribute/webapp/webapp-to-redux/